tests/testthat/test-geoknife_utils.R

```diff
@@ -58,6 +58,22 @@ test_that("result fails with failed job",{
   expect_error(result(failed.id), "processing is incomplete or has failed. See check()")
 })
 
+# test_that("percent complete works",{
```

Comment:
Do we need to `skip_on_cran()` here or something similar?

Reply:
we could include the xml responses from various stages in the processing (w/ different percent complete) in `inst/extdata` and use that for testing so that we don't hit the service. One issue would be if the service changes...

Reply:
With how these method calls work it's going to be hard to use a local file. Would need to create a package function of internal method that we could call from a test with geoknife::: I've done a bit of this on my testing branch. I'll look at this one now.

Reply:
This is doable, will add a bit of logic here: https://github.com/USGS-R/geoknife/blob/master/R/check.R#L45 and contribute this to my testing PR later today or this week.

Reply:
30 minutes of putzing has me thinking we should do this as part of #363 

I'm having a hard time getting a local file injected instead of a URL and gcontent has where I'm running into a hard time. Will leave a note over there.